cmd/flightctl-agent/main.go

```diff
@@ -47,6 +47,12 @@ func NewAgentCommand() *agentCmd {
 		a.log.Fatalf("Error validating config: %v", err)
 	}
 
+	logLvl, err := logrus.ParseLevel(a.config.LogLevel)
+	if err != nil {
+		logLvl = logrus.ErrorLevel
```

Comment:
Thanks for putting up this PR just curious why not info?

Reply:
You are right, Info is the default we would want. Updated